packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -142,31 +106,10 @@ class CupertinoApp extends StatefulWidget {
        assert(debugShowCheckedModeBanner != null),
        super(key: key);
 
-  /// The widget for the default route of the app ([Navigator.defaultRouteName],
-  /// which is `/`).
-  ///
-  /// This is the route that is displayed first when the application is started
-  /// normally, unless [initialRoute] is specified. It's also the route that's
-  /// displayed if the [initialRoute] can't be displayed.
-  ///
-  /// To be able to directly call [MediaQuery.of], [Navigator.of], etc, in the
-  /// code that sets the [home] argument in the constructor, you can use a
-  /// [Builder] widget to get a [BuildContext].
-  ///
-  /// If [home] is specified, then [routes] must not include an entry for `/`,
-  /// as [home] takes its place.
-  ///
-  /// The [Navigator] is only built if routes are provided (either via [home],
-  /// [routes], [onGenerateRoute], or [onUnknownRoute]); if they are not,
-  /// [builder] must not be null.
-  ///
-  /// The difference between using [home] and using [builder] is that the [home]
-  /// subtree is inserted into the application below a [Navigator] (and thus
-  /// below an [Overlay], which [Navigator] uses). With [home], therefore,
-  /// dialog boxes will work automatically, the [routes] table will be used, and
-  /// APIs such as [Navigator.push] and [Navigator.pop] will work as expected.
-  /// In contrast, the widget returned from [builder] is inserted _above_ the
-  /// [CupertinoApp]'s [Navigator] (if any).
+  /// {@macro flutter.widgets.widgetsApp.navigatorKey}
+  final GlobalKey<NavigatorState> navigatorKey;
+
+  /// {@macro flutter.widgets.widgetsApp.home}
   final Widget home;
```

Comment:
For all of these properties that seem to be replicas of WidgetApp, would it make sense to expose a virtual property that simply returns whatever is in the WidgetApp?  Replicating a bunch of properties seems like it could only ever create maintenance overhead.  Is there any other reason to replicate?